### ISSUE TITLE
How to create a slider/seeker? 

### ISSUE THREAD

**Issue by manjunath-nuveb** [bamlab/react-tv-space-navigation#40]
**Is your feature request related to a problem? Please describe.**
I am building custom controls for my video player. how to build a seeker using slider?

**Comment by manjunath-nuveb** [comment]
closing for now

**Comment by pierpo** [comment]
Cool! When I said "it's ok" I thought it was very unlikely to have elements on the side 😂 
How will you trap the focus?

**Comment by ionictest2017** [comment]
hey @pierpo @manjunath-nuveb  I have an issue, related to managing the focus on the video player
actually, I have the main container with a focus after clicking on the main container an overlay opens with controls. there should be a main container focus disable and after hiding (after 15s auto-hide) overlay of buttons(controls) all controls focus should be disabled and the main container focus should be enable. but I am facing issue to do it

**Comment by justinbeatz** [comment]
Hi all, sorry for posting this in here, I didn't wanted to create a new similar issue, I'm having kind the same issue as @ionictest2017 and I want to handle the remote presses to show the player controls if is hide, then if I press back hide the controls and if I press back again to go back to the list (I hope you understand). I checked what @pierpo poster [here](https://github.com/bamlab/react-tv-space-navigation/issues/40#issuecomment-1842573573)  but I could not find a way to get the useRemoteControl hook from the library (I checked the source code and there is a useRemoteControl hook with a different signature: [useRemoteControl Hook](https://github.com/bamlab/react-tv-space-navigation/blob/main/packages/lib/src/spatial-navigation/hooks/useRemoteControl.ts)).

Did someone have a workaround for handling the keyboard in multiples scenarios?

Note: I'm using the code from the example with some modifications to handle the keyboard in the whole app: [Keyboard Components](https://github.com/bamlab/react-tv-space-navigation/tree/main/packages/example/src/components)
